claude-ops/tests/outbound-guard/test-broken-alias.py

```
@@ -0,0 +1,133 @@
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+SRC = os.path.abspath(os.path.join(HERE, "..", "..", "scripts", "outbound-guard"))
+HOOK = os.environ.get("OUTBOUND_HOOK", os.path.join(SRC, "block-outbound-comms.py"))
+# The approval store sits at a fixed path shared by every guard, so a test that leaves
+# credit in it changes the result of whichever suite runs next.
+STATE_FILES = ("/tmp/.claude-outbound-guard.json", "/tmp/.claude-send-ok")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/.claude-outbound-guard.json"

(S108)